project/Dependencies.scala

```diff
@@ -12,24 +12,24 @@ import sbt._
 object Dependencies {
   // keep in sync with .github/workflows/unit-tests.yml
   val scala212Version = "2.12.18"
-  val scala213Version = "2.13.11"
-  val scala3Version = "3.3.0"
+  val scala213Version = "2.13.12"
+  val scala3Version = "3.3.1"
   val scalaVersions = Seq(scala212Version, scala213Version, scala3Version)
 
-  val pekkoVersion = System.getProperty("override.pekko.version", "1.0.1")
+  val pekkoVersion = System.getProperty("override.pekko.version", "1.0.2")
   val pekkoVersionInDocs = "1.0"
   val cassandraVersionInDocs = "4.0"
 
   // Should be sync with the version of the driver in Pekko Connectors Cassandra
-  val driverVersion = "4.15.0"
-  val driverVersionInDocs = "4.14"
+  val driverVersion = "4.17.0"
```

Comment:
Can we put the cassandra driver update in a separate PR? I think the Scala and Pekko upgrades could be backported to 1.0.x branch.

Reply:
sure

Reply:
moved to #121